openapi/src/parts/controlled_gcp_gce_instance.yaml

```diff
@@ -222,6 +222,12 @@ components:
             of 64000 GB (64 TB). If not specified, this defaults to 500.
           type: integer
           format: int64
+        bootDiskSizeGb:
```

Comment:
Added optional `bootDiskSizeGb` parameter, it was previously hardcoded to 100G. The Leo image is slightly bigger.

Reply:
Should this be required? What is the behavior if null is passed to the builder params?

Reply:
If null, the GCP behavior is to default to the size of the source image.

I did discover that the `vmImage` parameter is required, though WSM doesn't treat it as such. I can do a pass to mark that required, and update the swagger documentation. (good call)